app/views/plans/add_operation_period.js.erb

```diff
@@ -1,4 +1,7 @@
-$(".tabs-content").append("<%= j render(partial: 'operation_period', locals: { count: @count } )%>")
+<% plan_form = form_builder_for(@plan) %>
+<% 1.upto(@count - 1).map { |c| render(partial: 'operation_period', locals: { plan_form: plan_form, count: c } ) } %>
```

Comment:
This is pretty hacky; it makes sure the id counter in the new form is the right one based on how many existing operation periods there are.